### PR TITLE
Add README docs for "replan" and "restart"

### PR DESCRIPTION
**Pull request by benhoyt**
Add a section to the README about "replan" and "restart".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,31 @@ $ pebble stop srv1        # stop one service
 
 When stopping a service, Pebble sends SIGTERM to the service's process group, and waits up to 5 seconds. If the command hasn't exited within that time window, Pebble sends SIGKILL to the service's process group and waits up to 5 more seconds. If the command exits within that 10-second time window, the stop is considered successful, otherwise `pebble stop` will exit with an error.
 
+### Updating and restarting services
+
+When you add update service configuration (by adding a layer), the services changed won't be automatically restarted. To restart them and bring the service state in sync with the new configuration, use `pebble replan`.
```
Comment (hpidcock) [inline README.md:200]: When you _add update_ service configuration??

File: README.md [hunk 2]
```diff
@@ -195,6 +195,31 @@ $ pebble stop srv1        # stop one service
...
+
+The "replan" operation restarts `startup: enabled` services whose configuration has changed between when they started and now; if the configuration hasn't changed, replan does nothing. Replan also starts `startup: enabled` services that have not yet been started.
```
Comment (hpidcock) [inline README.md:202]: ~has~ _have_ changed

**Review by hpidcock — APPROVED** [review]
A few items to reconsider.

File: README.md (hunk 2 above)
Comment (benhoyt) [reply README.md]: Fixed

**Review by jnsgruk — APPROVED** [review]

**Pull request merged**